src/components/nve-icon-button/nve-icon-button.component.ts

```diff
@@ -0,0 +1,45 @@
+import { html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { INveComponent } from '@interfaces/NveComponent.interface';
+import styles from './nve-icon-button.styles';
+import '../nve-icon/nve-icon.component';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import NveButton from '../nve-button/nve-button.component';
+
+/**
+ * Knapp som viser kun ikon. Den støtter de samme egenskapene som en vanlig nve-button.
```

Comment:
I stedet for "støtter" kan du skrive "har". Det er litt lettere å lese, tror jeg